README.md

```diff
@@ -1,24 +1,209 @@
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database initialization
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📖 Cost Commander <a name="about-project"></a>
 
-* Deployment instructions
+**Cost Commander** is a full stack app created with ruby on rails and postgresql
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Back-end</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- **[It is a website for tracking the expecnes]**
+- **[It has a database included on real time]**
+- **[It uses Ruby on rails for the front end and back end]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://rails-i12r.onrender.com/)
+- [ video tutorial ](https://drive.google.com/file/d/1sLPj2u79EM7wvpPxlYuQqlg7NYwYgWH9/view?usp=sharing)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need :
+Install this:
+Ruby
+Rails
+Visual Studio Code
+Git
+Postgresql
+
+
+### Setup
+
+Clone this repository to your desired folder:
+and a open a new terminal on your visual studio code  
+
+### Install
+
+Install this project with:
+bundle install
+
+### Usage
+
+To run the project, execute the following command:
+Go to the file called /config/database.yml 
+and change the name, password and host of the database
+use the proper information from your local instalation of postgresql
+
+then run:
+
+rails db:create db:migrate
+
+then run:
+
+rails S
+
+
+### Run tests
+
+To run tests, run the following command:
+rspec
+
+### Deployment
+
+You can deploy this project using:
+you can use any service avalaible on internet such as:
+
+Render
+Heroku
+
+I recomend Render since is easy to use
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] Please remove this line as it is not supposed to include unnecessary directions.